tuxemon/states/combat/combat.py

```diff
@@ -713,47 +714,54 @@ def add_monster_into_play(
                 (partial(self.alert, message), 0)
             )
 
+    def get_status_ico(
```

Comment:
Maybe call it `get_status_icon_position`?

Reply:
ok